src/Parser/ParseAmount.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+
+namespace Silarhi\Cfonb\Parser;
+
+
+use Silarhi\Cfonb\Exceptions\ParseException;
+
+class ParseAmount
```

Comment:
Could you please rename `ParseAmount` class to `AmountParser`?

Reply:
done